tutorials/io/binary_serialization_api.rst

```diff
@@ -385,8 +385,29 @@ Every name string is padded to 4 bytes.
 16: :ref:`RID<class_rid>` (unsupported)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-17: :ref:`Object<class_object>` (unsupported)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+17: :ref:`Object<class_object>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++----------+-------+-----------+---------------------------------------------------------------------+
+| Offset   | Len   | Type      | Description                                                         |
++==========+=======+===========+=====================================================================+
+| 4        | 4     | Integer   | val&0x7FFFFFFF = properties, val&0x80000000 = shared (bool)         |
++----------+-------+-----------+---------------------------------------------------------------------+
+
+Then what follows is, for amount of "properties", pairs of name and value,
+one after the other, using this same format.
```

Comment:
Every case should be described appropriately according to [the source code](https://github.com/godotengine/godot/blob/master/core/io/marshalls.cpp#L1178-L1241). Like what is stored depending on `p_allow_objects` flag and whether object is valid or not.

Reply:
Thanks, good point! Done.